### PR TITLE
feat(devtools-view): Add missing info tooltips

### PR DESCRIPTION
**Pull request by Josmithr**
Adds info tooltip badges for `scopes` and `mode` Client properties.

image

image

### CODE REVIEW

File: packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx [hunk 1]
```diff
@@ -37,3 +37,23 @@ export const containerStatusTooltipText = (
 		.
 	</div>
 );
+
```
Comment (Josmithr) [inline packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx:40]: Looking for feedback on the tooltip contents here. I don't have a strong understanding of the semantics / implications of these beyond what is in the code comments.

Additionally, if we have other public documentation on these concepts, I am not aware of them. But I would love to link them if we do.

**Comment by seanimam** [comment]
Only comment I have is in regards to the text sizing in the tooltips and how does it look, interact with different window sizes. (There's no font size set so idk how it determines what sizing to use)

I have setup hooks in previous front-end projects that essentially keep state and tell you what size the screen is based on preconfigured breakpoints which you can use to resize text e.g.
```
const isSm = useResponsiveWindowSize(<breakpoint>, <up/down/exactly/etc>)
```

We may find this useful as we dig into making our ui more responsive

**Review by seanimam — APPROVED** [review]

File: packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx (hunk 1 above)
Comment (alexvy86) [reply packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx:40]: Client mode matches my understanding. If anything, I'd add `Note that the current connection mode does not necessarily reflect the client's permissions. A client with write permissions can connect to a container in view-only mode if it wishes.`.

The scopes piece I'm not familiar with, so can't comment there, but the current content feels ok.

File: packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx [hunk 2]
```diff
@@ -37,3 +37,23 @@ export const containerStatusTooltipText = (
...
+/**
+ * Description of the Client's "Mode" property in the audience view.
+ */
+export const clientModeModeTooltipText = (
```
Comment (alexvy86) [inline packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx:44]:
```suggestion
export const clientModeTooltipText = (
```

**Review by alexvy86 — APPROVED** [review]

**Pull request merged**